packages/angular/build/src/utils/server-rendering/prerender.ts

```diff
@@ -275,8 +288,12 @@ async function getAllRoutes(
     recordTiming: false,
   });
 
+  const errors: string[] = [];
   const { routes: extractedRoutes, warnings }: RoutersExtractorWorkerResult = await renderWorker
     .run({})
+    .catch((err) => {
+      errors.push(`An error occurred while extracting routes.\n\n${err.stack}`);
```

Comment:
In this case, I opted to display the stack trace instead of just the error message. This can help users identify the exact part of their application that is causing the error.

Alternatively, I can modify this behavior to only display the stack trace when verbose mode is enabled.